ui/src/styles/overrides.css

```diff
@@ -17,6 +17,10 @@ div.va-modal__title {
   font-size: initial;
 }
 
+div.va-modal--fixed-layout div.va-modal__header div.va-modal__title {
+  padding-left: 0;
```

Comment:
The current version of Vuestic modal, when using fixed-layout prop, adds an extra left padding to the va-modal__title class.